public/js/get_events.js

```diff
@@ -2,10 +2,12 @@
 var getEventsModule = function (queryParams, callback) {
   var baseUrl = 'https://nazareth-open-tourism-platform.herokuapp.com/events';
   var url = '';
-  if (!queryParams.date_from || !queryParams.date_to) {
+  if (!queryParams.date_from && !queryParams.date_to) {
     url = baseUrl;
-  } else {
+  } else if (queryParams.date_from && queryParams.date_to) {
     url = baseUrl + '/?date_from=' + queryParams.date_from + '&date_to=' + queryParams.date_to;
+  } else if (queryParams.date_from && !queryParams.date_to) {
+    url = baseUrl + '/?date_from=' + queryParams.date_from;
```

Comment:
This needs refactoring - a situation where you have `queryParams.date_to` and no `queryParams.date_from` would not do anything